server/index.js

```diff
@@ -20,114 +21,184 @@ app.use(bodyParser.urlencoded({extended: true}));
+//   }, function(err) {
+//       console.log(err.message);
+//   });
+// })
 //
 //update existing data by room ID
-app.put('/api/rooms/:id', (req,res) =>{
-  let id = req.params.id;
-  client.update({
-    "id" : id,
-    "index" : "rooms",
-    "body" : {
-      "doc" : {
-      "listing" : {
-          "cost" : 4,
-          "rating" : 5,
-          "childrenChosen" : 3
-        }
-      }
-    }
-  }).then(function(resp, status) {
-    console.log(resp);
-    res.send(status);
-  }, function(err) {
-    console.log(err.message);
-  })
-})
+// app.put('/api/rooms/:id', (req,res) =>{
+//   let id = req.params.id;
+//   client.update({
+//     "id" : id,
+//     "index" : "rooms",
+//     "body" : {
+//       "doc" : {
+//       "listing" : {
+//           "cost" : 4,
+//           "rating" : 5,
+//           "childrenChosen" : 3
+//         }
+//       }
+//     }
+//   }).then(function(resp, status) {
+//     console.log(resp);
+//     res.send(status);
+//   }, function(err) {
+//     console.log(err.message);
+//   })
+// })
 //
 //delete a record
-app.delete('/api/rooms/:id', (req, res) => {
-  let id = req.params.id;
-  client.delete({
-    "id": id,
-    "index" : 'rooms',
-  }).then(function(err, resp, status) {
-    res.send(status);
-  }, function(err, resp,) {
-    console.log(err.message);
-  });
-})
+// app.delete('/api/rooms/:id', (req, res) => {
+//   let id = req.params.id;
+//   client.delete({
+//     "id": id,
+//     "index" : 'rooms',
+//   }).then(function(err, resp, status) {
+//     res.send(status);
+//   }, function(err, resp,) {
+//     console.log(err.message);
+//   });
+// })
 //
 //
 ////////////////////////////////////////////////////////////
 
 
 
+///////////////////////////////////////////////////////////
+//
+// **           API ENDPOINTS FOR POSTGRES            **
+//
+///////////////////////////////////////////////////////////
+//
+// add a listing
+app.post('/api/listing/:id', (req, res) => {
+  let id = req.params.id
+  const { perNight, Rating, RatingAmount, guestsAllowed, guestsInfants, cleaningFee, serviceFee, occupancyFee, daysMinimum } = request.body
+
+  pool.query(`INSERT INTO lissting (ID, perNight, Rating, RatingAmount, guestsAllowed, guestsInfants, cleaningFee, serviceFee, occupancyFee, daysMinimum) VALUES (${id}, ${perNight}, ${rating}, ${ratingAmount}, ${guestsAllowed}, ${guestsInfants}, ${cleaningFee}, ${serviceFee}, ${occupancyFee}, ${daysMinimum})`, (error, results) => {
+    if (error) {
+      throw error
+    }
+    res.status(201).send(`Listing added with ID: ${results.ID}`)
+  })
+})
+//
+// get a listings by ID
+app.get('/api/listing/:id', (req, res) => {
+  let id = req.params.id
+  pool.query('SELECT * FROM listing WHERE id = $1', [id], (error, results) => {
+    if (error) {
+      throw error
+    }
+    res.status(200).json(results.rows)
+  })
+}
+// get a listings reservations by ID
+get('/api/listing/:id/reservations')
+app.get('/api/listing/:id', (req, res) => {
```

Comment:
Confused with Line 163 and 164, not sure if its a chain or not commented out or copy paste wrong, but line 164 should be app.get('/api/listing/:id/reservations')?